src/test/functional/steps/court-addresses.ts

```diff
@@ -0,0 +1,156 @@
+import {Then, Given, When} from 'cucumber';
+import {expect} from 'chai';
+
+import * as I from '../utlis/puppeteer.util';
+
+Then('I click the Addresses tab', async () => {
+  const selector = '#tab_addresses';
+  const elementExist = await I.checkElement(selector);
+  expect(elementExist).equal(true);
+  await I.click(selector);
+});
+
+Given('I will make sure to clear all entries of the primary address', async () => {
+  const selectorPrimaryAddressLines = '#primaryAddressLines';
```

Comment:
Could possibly make this a method that takes a string, checks the element, and then clears the field, like: 

```
async function checkAndClearAddressField(addressFieldElement: string {
  expect(await I.checkElement(addressFieldElement)).equal(true);
  await I.clearField(addressFieldElement);
}
```

Then something like: 

Given('I will make sure to clear all entries of the primary address', async () => {

await fillInUsernameAndPassword('#primaryAddressLines');
await fillInUsernameAndPassword('#primaryAddressWelsh');
await fillInUsernameAndPassword('#primaryAddressTown');
await fillInUsernameAndPassword('#primaryAddressTownWelsh');
await fillInUsernameAndPassword('#primaryAddressPostcode');
}

Reply:
I've created the method.